cookbooks/ros2_windows/recipes/pip_installs.rb

```diff
@@ -36,9 +35,16 @@
   "jazzy" => "pyparsing==3.1.1",
   "rolling" => "pyparsing==3.1.1",
 }
-
 required_pip_packages << pyparsing_dependency[node["ros2_windows"]["ros_distro"]]
 
+pyqt5_sip_dependency = {
+  "humble" => "PyQt5-sip==12.9.1",
+  "iron" => "PyQt5-sip==12.9.1",
+  "jazzy" => "PyQt5-sip==12.13.0",
+  "rolling" => "PyQt5-sip==12.13.0",
```

Comment:
Why are these different?

Haven't we already shipped binary archives for humble and iron patch releases using the "latest" version?

Reply:
While that's true, since we haven't been pinning it at all it means that we've likely shipped different versions of it for different patch releases of Humble (and probably Jazzy).  I can't tell for sure since we don't keep the logs around for old patch releases, but given that Humble was released in May 2022 and PyQt5-sip has had a number of releases (https://pypi.org/project/PyQt5-sip/#history), I don't think this is a huge problem.

What I was trying to do here was to pin this to the equivalent version that is used in Ubuntu (22.04 for Humble and Iron, 24.04 for Jazzy and Rolling).